### DISCUSSION TITLE
AAD i nie tylko (pierwsze kroki)

### DISCUSSION THREAD

**Discussion by AdamS90** [AzureCommunityPL/group#109] [Pomoc]
Cześć, poniżej mam do zrealizowania kilka punktów, które nie wiem jak ugryźć. Mogę sie logować do portalu azure, nie mam w nim subskrypcji i nie wiem skąd wziąć wiedzę by rozwiązać poniższe podpunkty, a gdy próbuję sam to ciągle trafiam na jakieś komplikacje. Już przechodzę do tego co muszę zrobić (i w czym bym prosił o wsparcie):

1. Prosze utworzyć dzierżawę office 365 (lub wykorzystac istniejaca) z wlasna domena zakladajac, ze organizacja korzysta z uslug exchange online, skype for business oraz mdm.
2. Proszę określić schemat nazewniczy dla kont uzytkownikow.
3. Proszę określić schemat nazewniczy dla wszystkich rodzajow grup stosujacych w ich nazwach wybrany przez siebie prefiksu lub sufiksu. 
4. Prosze zastosowac funkcje polityk nazewniczych grup Microsoft 365 w Azurze AD narzucajac ustawienia dla wybranego sufiksu lub prefiksu. 
5. Prosze okreslic i ustawic polityke wygasania grup Microsoft 365 w Azurze AD. 
6. Prosze okresic schemat nazewniczy dla urzadzen. 
7. Prosze utworzyc konta tożsamości dla organizacji, która sklada sie z komorek: zarzad oraz departamenty kadr, ksiegowosci, informatyki i logistyki. Struktura musi zawierac 20 uzytkownikow, dowolnie przydzielonych do poszczególnych komorek organizacji. Prosze uzupelnic u uzytkownikow atrybut Department oraz przydzielic kazdego uzytkownika do jednostki administracyjnej. 
8. Prosze o przygotowanie graficznego schematu organizacyjnego przedsiebiorstwa. 
9. Prosze zainstalowac moduly PS pozwalające na zarzadzanie Teams, SharePoint, Skype for Business, Azure AD oraz Exchange Online.

Dopisze jeszcze, ze jako poczatkujacy w Azure poszukuje kogos do korepetycji z tych tematów, wsparcia w realizowania różnych zagadnień (jak np. powyżej), wskazywania skąd wiedzę nabyc w tym zakresie.
Pewnie spotkania by byly przez Teamsy.

**Comment by kamil-mrzyglod** [comment]
Trochę trolluję, ale to są jakieś pytania na egzamin / zadania na sprint? ;) Bo brzmi jak coś, co mógłbyś komu przedstawić aby Ci wycenił i zrealizował za $$$.

**Reply by AdamS90** [reply]
To jest cos co musze sam wykonac i na podstawie tego jeszcze utworzyc sprawozdanie. Poki co 1szy punkt mnie hamuje by móc isc dalej.

**Reply by kamil-mrzyglod** [reply]
To rzecz pierwsza - nie wiadomo co Ciebie blokuje w tym punkcie? Jakiś specyficzny błąd? Czy nie wiesz skąd wziąć rozwiązania? Generalnie instrukcji w necie jak tworzyć / obsługiwac tenanta AAD / O365 jest dość sporo...

**Reply by AdamS90** [reply]
Nie wiem jak to wykonać. Co to znaczy utworzyć dzierzawe office 365? Czy do tego musze miec subskrypcję w Azurze? W co wejsc by to poustawiać itd. Brak mi czegos, jakiegos źródła wiedzy na którym bym sie mogl oprzec, ktore bym mogl poczytac, zglebic i wykonac z czasem. Stad tez szukam kogos do korepetycji by mi pomogl sie tego nauczyc. Polecono mi na githubie zalozyc temat to tak tez zrobilem.

**Reply by MarczakIO** [reply]
Podpisuje sie pod tym co Kamil napisal. Obecnie to troche wyglada jakbys poprosil kogos zeby zrobil z Toba (lub za Ciebie) projekt/inzynierke. 

Idac za zasada ze stackoverflow. Napisz dokladnie ktory punkt zaczales, jaki miales problem, co znalazles w danym temacie i jaki masz problem. 

Chyba ze faktycznie szukasz kogos kto po akademicku Cie tego nauczy od zera. To powiem Ci ze wedlug mnie to slaba droga, bo za friko watpie ze sie ktos tego podejmie, a za $$$ Ci sie to nie oplaci raczej.

**Reply by MarczakIO** [reply]
I zeby nie bylo let me google that for you are doslownie zaznaczylem co napisales i wkleilem w google
![image](https://user-images.githubusercontent.com/32310272/95740564-0f353780-0c8d-11eb-9cb3-2079288314e8.png)

**Reply by mgrabarz** [reply]
I uwaga ode mnie. 
Zamiast czytać o "dzierżawach" poczytaj o "tenantach". Czyli zamień pl-pl w adresie URL na "en-us". Tego tłumaczonego maszynowo bełkotu nie da się czytać. 🤪

**Reply by PawelPlawiak** [reply]
Na większość punktów widziałem odpowiedź w OneNote z zajęć ;)
